### PR TITLE
chore: update dependencies to `community-dev`

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
- uses `community-dev` for `agoric` dependencies instead of `beta`

- uses `ses@18.8.8` for node 18.19 compat 

- motivation: [drop node 16 EOL](https://github.com/Agoric/agoric-sdk/pull/8365) ci is [failing](https://github.com/Agoric/agoric-sdk/actions/runs/7687773301/job/20948202465?pr=8365#step:9:142)

### ~~To Do~~:

- ~~Update tests for `test-callSpread.js`. It's reliant on `setupBasicMints` and `zoeTestHelpers` which are ~~no longer~~ exported in `@agoric/zoe` `master` but not `upgrade-13` / `community-dev`~~.

This failure is only with `yarn install`, not `agoric install` which CI still uses.

```js
import { setup } from '@agoric/zoe/test/unitTests/setupBasicMints.js';
import { assertPayoutDeposit } from '@agoric/zoe/test/zoeTestHelpers.js';
```
```bash
  Uncaught exception in snippets/zoe/contracts/test-callSpread.js

  Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/0xpatrick/repos/agoric-documentation/node_modules/@agoric/zoe/test/unitTests/setupBasicMints.js' imported from /Users/0xpatrick/repos/agoric-documentation/snippets/zoe/contracts/test-callSpread.js

  ✖ snippets/zoe/contracts/test-callSpread.js exited with a non-zero exit code: 1
  ```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,26 +36,26 @@
...
+    "@agoric/cosmic-swingset": "community-dev",
+    "@agoric/ertp": "community-dev",
+    "@agoric/notifier": "community-dev",
+    "@agoric/solo": "community-dev",
+    "@agoric/vats": "community-dev",
+    "@endo/bundle-source": "2.5.2-upstream-rollup",
+    "@endo/eventual-send": "^0.17.2",
+    "@endo/init": "^0.5.56",
+    "@endo/promise-kit": "^0.2.56",
+    "@endo/ses-ava": "^0.2.40",
```
Comment (dckc) [inline package.json:58]: It's not obvious how these are related to community-dev.

Are they from package.json files from u13? Got a pointer?

File: package.json [hunk 2]
```diff
@@ -67,7 +67,7 @@
     "eslint-plugin-prettier": "^3.4.0",
     "import-meta-resolve": "^1.1.1",
     "prettier": "^1.19.1",
-    "ses": "^0.16.1",
+    "ses": "^0.18.8",
```
Comment (dckc) [inline package.json:70]: This doesn't seem related to community-dev. hm.

**Review by dckc — APPROVED** [review]
I'm a little fuzzy on some of the details.

I trust ci is checking the essential stuff.

File: package.json (hunk 1 above)
Comment (0xpatrickdev) [reply package.json:58]: I grabbed the versions from `yarn.lock`, lines 1323 - 1523:

https://github.com/Agoric/agoric-sdk/blob/5a6cdeb0c18ae9700d706445acf402f8d1e873c3/yarn.lock#L1328

https://github.com/Agoric/agoric-sdk/blob/community-dev/yarn.lock#L1328

File: package.json (hunk 2 above)
Comment (dckc) [reply package.json:70]: A separate commit might have been ideal but is probably not cost-effective now. This will change soon-ish anyway.

Thanks for the details here in the PR comments.
